packages/components/src/tag/tests/chromatic/TagList.stories.tsx

```diff
@@ -137,6 +137,24 @@ export const Readonly: TagListStory = {
     )
 };
 
+export const Validation: TagListStory = {
+    storyName: "validation",
+    render: () => (
+        <Stack>
+            <TagList validationState="invalid">
+                <Item key="mercury">Mercury Program</Item>
+                <Item key="gemini">Gemini Program</Item>
+                <Item key="apollo">Apollo Program</Item>
+            </TagList>
+            <TagList readOnly onClear={() => {}} validationState="invalid">
```

Comment:
third tag list with validationstate="valid"?

should be similar to not specifying it, but at least now it's tester